ts/Gantt/Pathfinder.ts

```diff
@@ -1081,6 +1081,11 @@ Pathfinder.prototype = {
         chart.series.forEach(function (series: Highcharts.Series): void {
             if (series.visible && !series.options.isInternal) {
                 series.points.forEach(function (point: Point): void {
+                    // For Gantt series the connect could be
+                    // defined as a dependency
+                    if (point.options && (point.options as Highcharts.GanttPointOptions).dependency) {
+                        (point.options as any).connect = (point.options as Highcharts.GanttPointOptions).dependency;
```

Comment:
Quite a lot castings here (to `any` and `GanttPointOptions`). Can we somehow improve this?